system-addon/content-src/components/Topics/_Topics.scss

```diff
@@ -4,36 +4,48 @@
 
   font-size: 13px;
   color: $topic-grey;
-  min-width: 780px;
-  line-height: 16px;
   margin-top: $topic-margin-top;
+  line-height: 1.6;
```

Comment:
I added the multiplier for the small sizes so that the text looks a little better wrapped. The 16px is for the full size